pkg/util/validation/limits.go

```diff
@@ -124,6 +126,8 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 
 	f.DurationVar(&l.RulerEvaluationDelay, "ruler.evaluation-delay-duration", 0, "Duration to delay the evaluation of rules to ensure the underlying metrics have been pushed to Cortex.")
 	f.IntVar(&l.RulerTenantShardSize, "ruler.tenant-shard-size", 0, "The default tenant's shard size when the shuffle-sharding strategy is used by ruler. When this setting is specified in the per-tenant overrides, a value of 0 disables shuffle sharding for the tenant.")
+	f.IntVar(&l.RulerMaxRulesPerRuleGroup, "ruler.max-rules-per-rule-group", 15, "Maximum number of rules per rule group per-tenant. 0 to disable.")
```

Comment:
**suggestion:** I think we should consider setting 0 as the default value for both new options. Otherwise this is a breaking change for some users. If we leave the default as is at least consider mentioning the new default value in the changelog.

Reply:
Great shout jacob - I've disabled them.